frontend/leaderboard/src/Main.re

```diff
@@ -28,6 +28,9 @@ let blocks =
        block.data.newBlock;
      });
 
+let blockHeight =
```

Comment:
This isn't actually a blockHeight stat but rather a count of the blocks in our data source, so can we update the name to `totalBlocks` and change the name in the spreadsheet to reflect this?

Reply:
That is true, good catch!